src/System/Envy.hs

```diff
@@ -312,7 +311,7 @@ setEnvironment' = setEnvironment . toEnv
 
 ------------------------------------------------------------------------------
 -- | Unset Environment from a `ToEnv` constrained type
-unsetEnvironment :: ToEnv a => EnvList a -> IO (Either String ())
+unsetEnvironment :: EnvList a -> IO (Either String ())
```

Comment:
I really should add a multi-ghc travis setup for this. I assume GHC 8 complained about it.